{{cookiecutter.repo_name}}/requirements/production.txt

```diff
@@ -1,7 +1,10 @@
 # Packages required in a production environment
 -r base.txt
 
-gevent==1.0.2
-gunicorn==19.3.0
-MySQL-python==1.2.5
+gevent==1.1rc1
```

Comment:
I'd prefer going that route to merging this and expecting new IDAs to use an unreleased version of two critical packages.